packages/stark-ui/src/modules/progress-indicator/directives/progress-indicator.directive.ts

```diff
@@ -88,6 +88,10 @@ export class StarkProgressIndicatorDirective implements OnInit, OnDestroy {
 	 * Finally, if the component should be hidden or shown, the stark-hide class is removed/added accordingly
 	 */
 	public ngOnInit(): void {
+		// TODO write a test for this
+		if (!this.starkProgressIndicator) {
+			throw new Error("[starkProgressIndicator] is required when using the starkProgressIndicator directive ");
```

Comment:
Can you add a unit test for this? And also, can you adapt the error message to align with the convention we follow for other errors: `StarkXXDirective: error message`.